functionalTest/test/specs/example.e2e.js

```diff
@@ -0,0 +1,81 @@
+const dashboard = require('../pageobjects/dashboard.page');
+const reusableObj = require('../utilities/reusableLibrary');
+
+
+describe('Covid-19 Dashboard', () => {
+    it('should have correct title', () => {
+        dashboard.open();
+        console.log("Url Launched....");
+        browser.pause(1000);
+        //  reusableObj.validateTitle('Covid-19 Dashboard');
+        //   expect(browser).toHaveTitle('Scottish COVID-19 Statistics');
+        title = 'Scottish COVID-19 Statistics';
+        expect(browser).toHaveTitle(title);        
+    });
+
+      it('should display Logo', () => {
```

Comment:
General comment - it's good to keep the formatting and syntax standard. In the code we use prettier in atom (or equivalents) with default formatting rules - it keeps the code tidy, and will stop unnecessary code churn as people change indentation, whitespace, etc. Here, the indentation is a bit off

Reply:
Implemented prettier formate in code.